main.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"os"
+
+	"github.com/bitrise-io/go-steputils/v2/stepconf"
+	"github.com/bitrise-io/go-utils/v2/command"
+	"github.com/bitrise-io/go-utils/v2/env"
+	. "github.com/bitrise-io/go-utils/v2/exitcode"
```

Comment:
Just leaving this placeholder comment because I have made a comment in your other PR for the same thing:
https://github.com/bitrise-steplib/bitrise-step-save-dart-cache/pull/1#discussion_r1101319169

Reply:
Fixed!